src/cgame/cg_ents.cpp

```diff
@@ -1318,6 +1318,8 @@ void CG_AddPacketEntities()
 
 	bool done[ MAX_GENTITIES ] = {};
 
+	unsigned highest = 0;
```

Comment:
Use `int` for everything in this commit. Since math does not work as expected if you go into negatives with unsigned numbers they should only be used if there is a good reason

Reply:
Done.